astropy/cosmology/parameter.py

```diff
@@ -45,16 +61,15 @@ class Parameter:
 
     _registry_validators = {}
 
+    @deprecated_renamed_argument("fmt", None, since="5.1")
```

Comment:
I've been flip-flopping. ``fmt`` is basically not used, since it prevents each dtype from specifying its own accuracy. OTOH it's already in the code and it's inoffensive.
I think I'd prefer to remove the deprecation commit, merge in this PR with the confusing similarity between ``fmt`` and ``format`` and tackle, in a followup PR, either renaming the argument or deprecating it.
Sound good?